README.md

```diff
@@ -1,181 +1,189 @@
 stream-php
```

Comment:
This is pedantic but we've added the following to the intro sections on the other API Clients:

"You can sign up for a Stream account at https://getstream.io/get_started."

e.g. https://github.com/getstream/stream-js

Reply:
yep, that's still here: https://github.com/GetStream/stream-php/blob/fix/readme/README.md#L10